boards/stk3700/include/periph_conf.h

```diff
@@ -245,14 +241,6 @@ static const uart_conf_t uart_config[] = {
         .cmu = cmuClock_UART0,
         .irq = UART0_RX_IRQn
     },
-    {
-        .dev = USART1,
-        .rx_pin = GPIO_PIN(PD, 1),
-        .tx_pin = GPIO_PIN(PD, 0),
-        .loc = USART_ROUTE_LOCATION_LOC1,
-        .cmu = cmuClock_USART1,
-        .irq = USART1_RX_IRQn
-    },
```

Comment:
no second UART on this board?

Reply:
No, because USART1 is used by `SPI_DEV(0)`, and that one is initialized during startup. That's why I removed it, because that would conflict. Same is true for STK3600.